### ISSUE TITLE
Wrong except option for nested entities

### ISSUE THREAD

**Issue by yoursdearboy** [ruby-grape/grape-entity#195]
Good day. I have a problem with except option. If I use it on entity Patient like this
```ruby
Patient::Entity.represent(patient, except: [testssets: [:patient]]).as_json
```
all works great, but I have an infinite recursion if use it on opposite entity Testsset
```ruby
Testsset::Entity.represent(testsset, except: [patient: [:testssets]]).as_json
```

I think that there's a problem with [Exposure::Base#serializeable_value](https://github.com/ruby-grape/grape-entity/blob/master/lib/grape_entity/exposure/base.rb#L61). All works fine in first case because it is one-to-many relation and handled by this code
```ruby
partial_output.map(&:serializable_hash)
```
In second case it is handled by this code (with options argument)
```ruby
partial_output.serializable_hash(options)
```
And lately options generated for a nested entity are rewritten in [Options#merge](https://github.com/ruby-grape/grape-entity/blob/master/lib/grape_entity/options.rb#L22) method like this
```ruby
options = {except: [:testssets]}
new_options = {except: [{patient: [:testssets]}]}
options.merge(new_options)
```
So, now we have except fields for a nested entity such as for the parent.
It seems that this code is broken and options hash should be passed in both cases. And except fields shouldn't be merged.

**Comment by dblock** [comment]
Indeed, it just wasn't implemented for nested except's, please contribute.
